src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.macios.cs

```diff
@@ -11,6 +11,7 @@ public void SetCount(int count)
 	{
 		UNUserNotificationCenter.Current.RequestAuthorization(UNAuthorizationOptions.Badge, (r, e) =>
 		{
+			Debug.WriteLine($"Error Requesting Authorization to Set Badge Count: {e.Description}");
```

Comment:
I'm 90% sure this will not be shipped on the release package... Maybe changing it to `Console.WriteLine` or using the `Trace` class that will have symbols for both Debug and Release?

https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.debug?view=net-7.0&redirectedfrom=MSDN

Reply:
Dang - you're totally right. 

It looks like a bunch of our services use `Debug.WriteLine`. 

Could you open a new Bug Report on this? 

I think the best move is to just merge this PR with the `Debug.WriteLine` included, and then open a new PR to fix all of them, including `AnimationBehavior`, `BaseConverter`, `ColorAnimationExtensions`, `StateContainer`, etc.